test/utils/image/manifest.go

```diff
@@ -382,7 +382,7 @@ func ReplaceRegistryInImageURL(imageURL string) (string, error) {
 	}
 
 	switch registryAndUser {
-	case "gcr.io/kubernetes-e2e-test-images":
+	case "k8s.gcr.io/e2e-test-images":
```

Comment:
same here. The ``cuda-vector-add:1.0`` image is the only image left that did not migrate to the new registry. I'm going to investigate more about it, see if it's really needed anymore, and what can we do about it.